movo_simulation/movo_gazebo_plugins/gazebo_grasp_plugin/src/GazeboGraspFix.cpp

```diff
@@ -400,7 +400,7 @@ void GazeboGraspFix::OnUpdate() {
         for (lIt=objIt->second.begin(); lIt!=objIt->second.end(); ++lIt){
             std::string linkName=lIt->first;
             CollidingPoint& collP=lIt->second;
-            gazebo::math::Vector3 avgForce=collP.force/collP.sum;
+            ignition::math::Vector3d avgForce=collP.force/collP.sum;
```

Comment:
What is this ignition library?

Reply:
so ros noetic uses a new version of gazebo. this new version of gazebo no longer defines the "math" types in gazebo::math. instead the types are found in the ignition lib